### ISSUE TITLE
Problem with the keyboard - HW kbd: Failed to set (null) as keyboard focus

### ISSUE THREAD

**Issue by micuentadecasa** [jamesmontemagno/MediaPlugin#102]
## Bug 

Version Number of Plugin: 2.3.0
Device Tested On: Ipad Mini 2 and Iphone 4
Simulator Tested On: 

### Expected Behavior
pick a photo
fill an entry
save the photo

### Actual Behavior

- pick photo
- when clicking in the entry the app crash

I have an Entry in a xaml page and if I try to use the cross.media picker and then fill the entry I got a crash:

HW kbd: Failed to set (null) as keyboard focus

In some situations setting the focus in the entry works but at last I got a crash when I click on the Entry

another situation, after I pick a photo even if I close the page and open a new one if I click on an entry or picker it crash...

it seems it has to be related to the memory, 
- for example I had a listview with the images that are being saved, if I quit this listview it crash but it does it lastly.
- if I try it many times, it doesn't crash the first time but when I got some time testing it, so it seems that is filling the memory and once it is full it crash.

can you try to create a new sample project that 
- open a new page
- pick a photo
- close page
this many times to see if there is a problem with the memory...

**Comment by jamesmontemagno** [comment]
Are you disposing properly of the images that are returned? 

If you display a lot of items in a list and aren't handling compressing the issues then Android will crash with out of memory issue. 

2.6.0 handles compressing images better with new settings for you automatically.

**Comment by micuentadecasa** [comment]
My problem is mainly with Ipad and Iphone, in IOS simulator it works ok. I have tried also quitting the listviews. How I can try the new version 2.6.0? I mean do I have to download binaries and reference them?

**Comment by jamesmontemagno** [comment]
You can turn on the "pre-release" in Nougat to update them to the latest beta versions.
I would assume it probably doesn't have to do with the library but maybe something else somewhere. If you have a small sample I can test it out.

**Comment by micuentadecasa** [comment]
Thanks @jamesmontemagno , I will try it and will try to make a small sample.
